<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>FDA Report</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Month/Year Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QComboBox" name="month_comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>January</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>February</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>March</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>April</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>May</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>June</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>July</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>August</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>September</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>October</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>November</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>December</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="downloadReport_pushButton">
             <property name="text">
              <string>Download</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Month</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="year_comboBox">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>2010</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2011</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2012</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2013</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2014</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2015</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2016</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2017</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2018</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2019</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2020</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2021</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2022</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2023</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2024</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2025</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2026</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Compare Report</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Current Downloaded File:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="filePath_lineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="browser_pushButton">
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>. . .</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>File to Compare:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="compareReports_pushButton">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Compare Reports</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="downloadedFile_lineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="browser2_pushButton">
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>. . .</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
